lib/jxl/convolve.cc

```diff
@@ -858,7 +858,7 @@ class ConvolveT {
     const int64_t stride = in.PixelsPerRow();
     RunOnPool(
         pool, ybegin, yend, ThreadPool::SkipInit(),
-        [&](const int y, int /*thread*/) HWY_ATTR {
+        [&](uint32_t y, size_t /*thread*/) HWY_ATTR {
```

Comment:
You still lost a couple of consts here :)

Reply:
Thanks for spotting, I think they're now all back.